scalyr_agent/monitor_utils/k8s.py

```diff
@@ -411,10 +411,10 @@ def __init__(
         # generate a hash we can use to compare whether or not
         # any of the pod info has changed
         md5 = hashlib.md5()
-        md5.update(name)
-        md5.update(namespace)
-        md5.update(uid)
-        md5.update(node_name)
+        md5.update(name.encode("latin1"))
+        md5.update(namespace.encode("latin1"))
+        md5.update(uid.encode("latin1"))
+        md5.update(node_name.encode("latin1"))
```

Comment:
Hm, can any of those values contain unicode? If it can, we should use ``utf-8`` encoding to avoid surprises.

Reply:
The reason I went with "latin1" is because it is supposed to be equivalent to what `md5.update()` does in python2 when passed in a raw string. I'm not actually 100% sure if "latin1" is always the default or if it infers the encoding somehow, utf-8 is probably safer though, I hope.